src/main/scala/org/zalando/spearheads/innkeeper/oauth/OAuthService.scala

```diff
@@ -2,36 +2,36 @@ package org.zalando.spearheads.innkeeper.oauth
 
 import akka.actor.ActorSystem
 import akka.http.scaladsl.Http
-import akka.http.scaladsl.model.{ HttpRequest, Uri }
+import akka.http.scaladsl.model.{HttpRequest, Uri}
 import akka.stream.ActorMaterializer
-import akka.stream.scaladsl.{ FlattenStrategy, Source }
-import com.google.inject.{ Inject, Singleton }
+import akka.stream.scaladsl.{FlattenStrategy, Source}
+import com.google.inject.{Inject, Singleton}
 import com.typesafe.config.Config
 import org.slf4j.LoggerFactory
 import spray.json._
 
 import scala.concurrent.duration._
-import scala.concurrent.{ Await, ExecutionContext }
-import scala.util.{ Failure, Success, Try }
```

Comment:
That reformatting probably comes from different settings in IDEA and scalastyle-reformat-on-compile... Try to avoid such changes by configuring one to be the same as the other